webui/src/pages/repositories/repository/objects.jsx

```diff
@@ -3,15 +3,14 @@ import React, {useRef, useState} from "react";
 import {UploadIcon} from "@primer/octicons-react";
```

Comment:
Suggestion: Upload and Delete are disabled for protected branches.

Reply:
Opened an issue: https://github.com/treeverse/lakeFS/issues/2510